lib/config/fileconf.go

```diff
@@ -1502,6 +1502,7 @@ type Discovery struct {
 	DiscoveryGroup string `yaml:"discovery_group,omitempty"`
 	// PollInterval is the cadence at which the discovery server will run each of its
 	// discovery cycles.
+	// Default: 5m
```

Comment:
I don't have a lot of faith in us keeping this up to date. Should we just refer to the name of the const that holds the default instead?